core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -582,6 +589,80 @@ export class BaseUser {
   }
 
   /**
+   * This function takes a screenshot of the page.
+   * If there's no parameter for newPage, it checks this.page instead of newPage.
```

Comment:
You can drop this line, I added it to the arg description below.

Reply:
Done